spacelift/data_module.go

```diff
@@ -93,6 +108,11 @@ func dataModule() *schema.Resource {
 							Description: "GitHub Enterprise namespace of the stack's repository",
 							Required:    true,
 						},
+						"id": {
+							Type:        schema.TypeString,
+							Description: "ID of the GitHub Enterprise application",
```

Comment:
I think it should be "ID of the GitHub Enterprise integration" instead of "application"